xla/stream_executor/rocm/rocm_driver.cc

```diff
@@ -110,6 +111,31 @@ std::string ToString(hipError_t result) {
       OSTREAM_ROCM_ERROR(ContextAlreadyInUse)
       OSTREAM_ROCM_ERROR(PeerAccessUnsupported)
       OSTREAM_ROCM_ERROR(Unknown)  // Unknown internal error to ROCM.
+#if TF_ROCM_VERSION >= 60200
```

Comment:
Can we get rid of all of these if/endif macros and have the version defined in runtime, abstracted and used in branching?

Reply:
Unfortunately, I think this is not easily possible: since this is a compile-time decision. IUUC, these new error codes might not be defined in old ROCM runtime headers, and hence the compilation would fail if we try to build it with just runtime checks using branching.